src/domain/service.ts

```diff
@@ -458,6 +458,24 @@ export async function search(params: SearchParams, identity: Identity): Promise<
   };
 }
 
+export const searchCollaboratorApplications = async (identity: Identity) => {
+  // find all applications on which the logged-in user has collaborator access
+  // using ego token email matched against collaborator googleEmail
+  const apps = await ApplicationModel.find({
+    state: 'APPROVED',
+    'sections.collaborators.list.info.googleEmail': identity.tokenInfo.context.user.email,
+  });
+
+  return apps.map(
+    (app: ApplicationDocument) =>
+      ({
+        appId: `${app.appId}`,
```

Comment:
@anncatton 
do we need `institution` also? 
https://app.zeplin.io/project/5a69fad2bde6330a929157fd/screen/60ca5a69e82750bf39b200d1
I see there are two different institutions